### DISCUSSION TITLE
Can't download video

### DISCUSSION THREAD

**Discussion by levadia** [aclap-dev/video-downloadhelper#1165] [Help and Q&A]
### Your question

I have a license for the plugin in Chrome but it fails to recognize videos from https://turkish4d.com/videolib
Could you please help?
Plugin version is: 9.1.0.43
(Not sure if it's safe to copy the key here so I've removed it from the data)

### Addon & CoApp version and details

version: 9.1.0.43
target: google
channel: stable
lang: ru
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"    "}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 12
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.4037, Firefox 129.0.1 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

I visited the web site whose URL you gave.  It's in a language I don't recognize.  I told Firefox to translate the page, which it did.  It showed me a link at the top of the page that said `English Language`.  Well, seeing as English is my native tongue, I clicked that link expecting to go to a page of English content.  But as usual, I was making the egregious error of applying logic to the situation.  Instead of a page in English, of course it took me to a page in Russian.  Of course.  Why wouldn't it?  The page was this:

https://turkish4d.com/course/43

VDH happened to recognize something on that page so I downloaded it.

![#01](https://github.com/user-attachments/assets/30725034-1e3e-42cd-9b4c-00c19ab6495a)

That's a pitiful download speed.  But the download did complete & I did have a video.  It turned out that the video track consisted of the still image you can see in the player in the image I've just posted.  Not a real video.  Just a still picture.  The audio track was somebody jabbering away in a language I don't speak, presumably Russian.

As useless as this download was, it did complete successfully for me.  Perhaps this is yet another instance of Firefox working better than Chrome on a web site.  We hit these pretty regularly.  Also, it could be that the VDH beta for Firefox contains some bug fixes that have not yet made it into the Chrome general availability release.  We no longer have VDH betas for Chrome.

I'm sure I didn't pick a representative example of something to download.  But you didn't help me by pointing me to a specific video.  You should always give the URL of a video, not the URL of a web site.  When you don't give the URL of a video, you get me stumbling around choosing probably something irrelevant.  You might try again.

**Reply by GrampaWildWilly** [reply]
By the way, I support your idea of removing even the partial VDH license key that the VDH information contains.  I always remove that from anything I post.  The key, even if it is just a partial key, should not be in that information.  The license key is entirely irrelevant to debugging a problem.  Nobody should post even a part of their key.  So you did the right thing by removing it.

**Comment by Mattmjs** [comment]
The license doesn't have anything to do with it as far as detection goes but I see why it doesn't work. I don't think embedded YouTube
is the problem either. Look at what they provided with that url, those videos are not detected because they are not using a proper
format. I don't know how this could work in the software.

See what I've highlighted in the image. this is a playlist for the video.

![turkish4hd](https://github.com/user-attachments/assets/d1e01bdb-d69e-47e2-8d73-3e0be581624f)

**Reply by GrampaWildWilly** [reply]
Right.  I was just pointing out that the original poster removed the partial license key from the pasted information.  I wholeheartedly agree with doing that.  I always remove it whenever I post my information.  The license key, even a partial license key, should not be included in the `Copy information to clipboard` function.  To my way of thinking, Paul should change the copy function by removing that little bit of irrelevant information.  He should also remove the E-mail address.  Those things are not relevant to problem diagnosis.

As for what you're showing in the Network Monitor, I don't know what to say.  I went to a page, VDH recognized something, & it got downloaded.  No problem.
